src/main/java/com/minecolonies/coremod/tileentities/TileEntityColonyBuilding.java

```diff
@@ -247,6 +276,24 @@ public void readFromNBT(final NBTTagCompound compound)
             colonyId = compound.getInteger(TAG_COLONY);
         }
 
+        if (compound.hasKey(TAG_INVENTORY, 10))
+        {
+            itemHandler.deserializeNBT(compound.getCompoundTag(TAG_INVENTORY));
+        }
+        else
+        {
+            // Compatibility code
+            itemHandler.deserializeNBT(compound);
+        }
+
+        if (compound.hasKey(TAG_CUSTOM_NAME, 8))
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Assign this magic number 8 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)